index.html

```diff
@@ -10,9 +10,10 @@
     while (1) {
```

Comment:
どちらかというと、以下の方が一般的です。
```suggestion
    while (true) {
```

Reply:
trueに修正しました